unit-tests-java8/src/test/java/org/eclipse/collections/test/bag/immutable/sorted/ImmutableSortedBagImplTest.java

```diff
@@ -16,10 +16,11 @@
 import org.eclipse.collections.impl.block.factory.Comparators;
 import org.eclipse.collections.impl.test.junit.Java8Runner;
 import org.eclipse.collections.test.IterableTestCase;
+import org.eclipse.collections.test.NoDetectOptionalNullTestCase;
 import org.junit.runner.RunWith;
 
 @RunWith(Java8Runner.class)
-public class ImmutableSortedBagImplTest implements ImmutableSortedBagTestCase
+public class ImmutableSortedBagImplTest implements ImmutableSortedBagTestCase, NoDetectOptionalNullTestCase
 {
```

Comment:
Good suggestion. I put this to SortedIterableTestCase and removed from its subclasses.